README.md

```diff
@@ -1,82 +1,116 @@
-|                   | Implicit Type Conversion                                                            | :white_check_mark: | String/number to Date/Time/Timestamp and more                                                                                                                                                                            |
-|                   | Object mapping                                                                      | :white_check_mark: | supported since 0.4.6, slow and limited to simple data types                                                                                                                                                             |
-
-## Planed to be removed
-
-| Feature                        | Version | Remark                                           |
+[![GitHub release (latest SemVer including pre-releases)](https://img.shields.io/github/v/release/ClickHouse/clickhouse-java?include_prereleases&label=Latest%20Release)](https://github.com/ClickHouse/clickhouse-java/releases/)
+[![Sonatype Nexus (Snapshots)](https://img.shields.io/nexus/s/com.clickhouse/clickhouse-java?label=Nightly%20Build&server=https%3A%2F%2Fs01.oss.sonatype.org)](https://s01.oss.sonatype.org/content/repositories/snapshots/com/clickhouse/)
+[![GitHub milestone](https://img.shields.io/github/milestones/progress-percent/ClickHouse/clickhouse-java/16)](https://github.com/ClickHouse/clickhouse-java/milestone/4)
+[![GitHub release (by tag)](https://img.shields.io/github/downloads/ClickHouse/clickhouse-java/latest/total)](https://github.com/ClickHouse/clickhouse-java/releases/)
+
+<div align="center">
+    <img src="https://github.com/ClickHouse/clickhouse-js/blob/a332672bfb70d54dfd27ae1f8f5169a6ffeea780/.static/logo.svg" width="200px" align="center">
+    <h1>ClickHouse Java Client & JDBC Driver</h1>
+</div>
+
+Table of Contents
+* [About The project](#about-the-project)
+* [Important Updates](#important-updates)
+* [Installation](#installation)
+* [Client V2](#client-v2)
+  * [Artifacts](#artifacts)
+  * [Features](#features)
+  * [Examples](#examples)
+* [Client V1](#client-v1)
+  * [Artifacts](#artifacts-1)
+  * [Features](#features-1)
+  * [Examples](#examples-1)
+* [Documentation](#documentation)
+* [Contributing](#contributing)
+
+## About the Project
+
+This is the official Java Client and JDBC for ClickHouse Database (https://github.com/ClickHouse/Clickhouse).
+Java client is the base component and has own API for working with ClickHouse in a "direct" way. JDBC driver is
+a library implementing JDBC API 1.3 on top of the Java client.
+
+Currently, there is a new version of the Java Client in developer preview. It is going to replace the old one very soon.
+Source code of the new version is located in the `client-v2` directory and usage examples are in `examples/client-v2` directory.
```

Comment:
we should add something to prefer `client-v2` usage since it gets more attention from the team team